linux/tests/pipe_linux_ut/mock_pipe.c

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <errno.h>

```

Comment:
Suspect this is not needed. Probably iwyu will say so :-)

Reply:
Does iwyu run against Linux as well? Will find out soon I guess...

---
In reply to: [562201137](https://github.com/Azure/c-pal/pull/67#discussion_r562201137) [](ancestors = 562201137)